source/sink-connector/fundamentals/post-processors.txt

```diff
@@ -187,6 +187,17 @@ sink connector.
        | For information on configuration, see the
          :ref:`Renaming by Regular Expression Example <field-renamer-regex-example>`.
 
+   * - | **NullFieldValueRemover**
+     - | **Full Path:**
+
+       .. code-block:: none
+
+          com.mongodb.kafka.connect.sink.processor.NullFieldValueRemover``
+
+       | Removes all null fields from the sink record.
+       | For more information on configuration, see the
+         :ref:`Allow List and Block List Examples <sink-allowlist-blocklist-examples>`.
+
```

Comment:
I think this one more be more similar to `KafkaMetaAdder` since its like a one off implementation. We can remove this snippet linking to allow/block list examples.